events/jsonschema/io.mintmetrics.mojito/mojito_failure/jsonschema/1-0-0

```diff
@@ -9,17 +10,25 @@
   "type" : "object",
   "properties" : {
     "waveId" : {
-      "type" : "string"
+      "description" : "A canonical ID of the test used in cookies and in experiment in reports",
```

Comment:
```experiment in reports``` should this be just ```experiment reports```? 

You might have picked up a typo in our docs for this?

Reply:
Yes, you're right, this is a copy paste error. Now if I would just remember where I picked it up...